packages/medusa/src/services/order-edit.ts

```diff
@@ -328,6 +342,79 @@ export default class OrderEditService extends TransactionBaseService {
     return orderEdit
   }
 
+  async addLineItem(
+    orderEditId: string,
+    data: AddOrderEditLineItemInput
+  ): Promise<void> {
+    return await this.atomicPhase_(async (manager) => {
+      const lineItemServiceTx = this.lineItemService_.withTransaction(manager)
+
+      const orderEdit = await this.retrieve(orderEditId, {
+        relations: ["order", "order.region"],
```

Comment:
suggestion: we can avoid the extra join here
```suggestion
        relations: ["order"],
```